src/tiled/preferences.cpp

```diff
@@ -435,6 +440,15 @@ void Preferences::setPatron(bool isPatron)
     emit isPatronChanged();
 }
 
+void Preferences::setOpenLastFilesOnStartup(bool open)
+{
+    if (mOpenLastFilesOnStartup != open)
+    {
+        mOpenLastFilesOnStartup = open;
+        mSettings->setValue(QLatin1String("Startup/OpenLastFiles"), open);
+    }
```

Comment:
Please watch the code style here and follow the same structure as other setters:
```c++
    if (mOpenLastFilesOnStartup == value)
        return;

    mOpenLastFilesOnStartup = value;
    mSettings->setValue(QLatin1String("Startup/OpenLastFiles"), value);
```
(also I consider `value` a better name here than `open`)